apps/prairielearn/src/migrations/20231031163354_client_fingerprints__create.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE
+  client_fingerprints (
+    id BIGSERIAL PRIMARY KEY,
+    user_id BIGINT REFERENCES users (user_id),
+    session_id TEXT REFERENCES user_sessions (session_id),
```

Comment:
This is a bit confusing, but we should be referencing `user_sessions.id` here, not `user_sessions.session_id`. `id` is the primary key and is a `bigint`; `session_id` is a random string that we encode in our cookies to identify a session.

Reply:
I have updated to use `user_sessions.id` in commit [b135adf](https://github.com/PrairieLearn/PrairieLearn/pull/8798/commits/b135adfd92d740104813013c22b9b36e61ec23bd). Because currently we only have the encoded session ID exposed in the request, I have added a query to retrieve the `user_session.id` by querying `user_session`  with `session id`.